src/main/java/io/specto/hoverfly/junit/core/model/FieldMatcherDeserializer.java

```diff
@@ -0,0 +1,26 @@
+package io.specto.hoverfly.junit.core.model;
+
+import com.fasterxml.jackson.core.*;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import java.io.IOException;
+
+class FieldMatcherDeserializer extends JsonDeserializer<FieldMatcher> {
+    @Override
+    public FieldMatcher deserialize(JsonParser jsonParser, DeserializationContext context) throws IOException {
+
+        FieldMatcher matcher = null;
```

Comment:
Upgrade v1 to v2 schema, it is easier to just to have the POJO for the latest JSON hoverfly schema in this library.